SQL/New_patches/2023-04-03_Add_Column_to_issues_attachments_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE issues_attachments ADD COLUMN `date_deleted` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
Should this be "NOT NULL"? Since it's called "date_deleted" It seems this value should be null if the attachment has not been deleted.